smoke-tests/apps/JmxMetric/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/JmxMetricTest.java

```diff
@@ -6,6 +6,7 @@
 import static com.microsoft.applicationinsights.smoketest.EnvironmentValue.TOMCAT_8_JAVA_11;
 import static com.microsoft.applicationinsights.smoketest.EnvironmentValue.TOMCAT_8_JAVA_17;
 import static com.microsoft.applicationinsights.smoketest.EnvironmentValue.TOMCAT_8_JAVA_21;
+import static com.microsoft.applicationinsights.smoketest.EnvironmentValue.TOMCAT_8_JAVA_23;
```

Comment:
why is this missing JAVA_23_OPENJ9?

Reply:
this test has never run on (any versions of) OpenJ9

I suspect it's due to OpenJ9 exposing different JMX metrics, but looking into it now, and will at least add comment in this file explaining...

Reply:
comment added (below)